### DISCUSSION TITLE
Where is the ERD information of CouchDB?

### DISCUSSION THREAD

**Discussion by kwonwoo078** [Budibase/budibase#7102] [Help] [answered]
We are planning to install and operate two budibases for development and operation purposes.

So I'm going to copy the application made from the development budibase to the operating budibase.

However, we are planning to manually copy the CouchDB information without using the export and import functions.

So, if there is ERD information of CouchDB, I think it will be easy to work.

Or if there is a way to suggest a better way, please reply.

**Comment by mike12345567** [comment, marked as answer]
Hi @kwonwoo078 - the latest version of the Budibase CLI has a mechanism for exporting everything from one Budibase installation and importing it onto another. You can see the info about this in the PR for it (https://github.com/Budibase/budibase/pull/6584) - its a very new feature, we will be documenting it on our readme docs very soon!

In terms of ERD information, CouchDB is a NoSQL database and doesn't really support relational data structures, its just a document database after all! What is very good at is replicating, [docs here](https://docs.couchdb.org/en/3.2.0/replication/intro.html#:~:text=3.-,Replication%20Procedure,the%20documents%20to%20the%20destination.) - you can essentially configure live sync between two CouchDB instances if you desire, so this is another route you could look into.

**Comment by kwonwoo078** [comment]
@mike12345567  Thank you for your answer
